README.md

```diff
@@ -1,2 +1,130 @@
-# plugin-grunt-tasks
-Several tasks requires for our plugins
+# grunt-yoast-tasks
+
+> the plugin that contains all custom yoast tasks
+
+## Getting Started
+This plugin requires Grunt `~0.4.5`
+
+If you haven't used [Grunt](http://gruntjs.com/) before, be sure to check out the [Getting Started](http://gruntjs.com/getting-started) guide, as it explains how to create a [Gruntfile](http://gruntjs.com/sample-gruntfile) as well as install and use Grunt plugins. Once you're familiar with that process, you may install this plugin with this command:
+
+```shell
+npm install grunt-yoast-tasks --save-dev
+```
+
+Once the plugin has been installed, it may be enabled inside your Gruntfile with this line of JavaScript:
+
+```js
+grunt.loadNpmTasks( "grunt-yoast-tasks" );
+```
+
+## The "set-version" task
+
+### Overview
+In your project's Gruntfile, add a section named `yoast_tasks` to the data object passed into `grunt.initConfig()`.
+
+```js
+grunt.initConfig({
+    yoast_tasks: {
+        options: {
+            // Task-specific options go here.
+        },
+           your_target: {
+            // Target-specific file lists and/or options go here.
+        }
+    }
+});
+```
+
+### Options
+
+#### options.base
+Type: `String`
+Default value: `''`
+
+The json file base object for the target to be in.
```

Comment:
Please capitalize to `JSON`.

Also, this description seems unclear. Please elaborate more on what this 'target' is